jobs/candlepinDockerImagesJob.groovy

```diff
@@ -0,0 +1,31 @@
+import jobLib.rhsmLib
+
+String baseFolder = rhsmLib.candlepinJobFolder
+
+job("$baseFolder/candlepin-docker-images") {
+    previousNames('candlepin-docker-images')
+    description('Periodically builds and pushes docker images from master')
+    label('rhsm-jenkins-slave6')
```

Comment:
I believe we set it to slave6 at the time because that's the only one that worked at the time (that was the openstack packet size issue),  this SHOULD work on `label('docker')` now